documentation/endpoints/signups.md

```diff
@@ -50,6 +50,30 @@ Example Response:
 }
 
 ```
+## Retrieve a specific signup
+
+```
+GET /api/v3/signups/:signup_id
```

Comment:
Should we offer the option to include posts under the signup?

Reply:
oh yeah I can totally do that and write a test for it as well !